tests/sentry/interfaces/test_stacktrace.py

```diff
@@ -401,7 +426,7 @@ def test_get_hash_ignores_sun_java_generated_methods(self):
         )
         result = interface.get_hash(platform='java')
         self.assertEquals(result, [
-            'sun.reflect.GeneratedMethodAccessor',
+            'sun.reflect.GeneratedMethodAccessor<auto>',
```

Comment:
I just realized this will change (break) existing grouping. It could effect a lot of Java issues that already exist. Maybe we should have `sun.reflect.GeneratedMethodAccessor` remain plain (not add `<auto>`) and make a not of this for hopefully improving in the future?